test/sequential/test-dgram-pingpong.js

```diff
@@ -1,30 +1,19 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var Buffer = require('buffer').Buffer;
-var dgram = require('dgram');
-
-var debug = false;
-var tests_run = 0;
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
 
 function pingPongTest(port, host) {
-  var callbacks = 0;
-  var N = 500;
-  var count = 0;
+  let pingsReceived = 0;
+  let pongsReceived = 0;
+  let done = false;
 
-  var server = dgram.createSocket('udp4', function(msg, rinfo) {
-    if (debug) console.log('server got: ' + msg +
-                           ' from ' + rinfo.address + ':' + rinfo.port);
+  const N = 5;
 
-    if (/PING/.exec(msg)) {
-      var buf = new Buffer(4);
-      buf.write('PONG');
-      server.send(buf, 0, buf.length,
-                  rinfo.port, rinfo.address,
-                  function(err, sent) {
-                    callbacks++;
-                  });
-    }
+  const server = dgram.createSocket('udp4', function(msg, rinfo) {
```

Comment:
I don't think we can on this one because there's no exact number of times that the callback will fire, just a minimum. That's what the `assert(pingsReceived >= N);` checks.